scripts/generate-babel-types-docs.js

```diff
@@ -37,6 +37,10 @@ function getType(validator) {
     return validator.type;
   } else if (validator.oneOfNodeTypes) {
     return validator.oneOfNodeTypes.join(' | ');
+  } else if (validator.oneOfNodeOrValueTypes) {
+    return validator.oneOfNodeOrValueTypes.join(' | ');
+  } else if (validator.oneOf) {
+    return validator.oneOf.map(val => util.inspect(val)).join(' | ');
```

Comment:
It's only run as part of `make build-dist`. I wasn't sure which stage you wanted it to run as?  I didn't mean to add an arrow function but I'm on node v6 so it just works for me :)